soh/soh/Enhancements/randomizer/ShuffleCows.cpp

```diff
@@ -72,6 +74,7 @@ void Rando::StaticData::RegisterCowLocations() {
     locationTable[RC_JABU_JABUS_BELLY_MQ_COW] = Location::Base(RC_JABU_JABUS_BELLY_MQ_COW, RCQUEST_MQ,   RCTYPE_COW,                              ACTOR_EN_COW, SCENE_JABU_JABU,         0x00,                               "MQ Cow",            RHT_JABU_JABUS_BELLY_MQ_COW, RG_MILK, SpoilerCollectionCheck::RandomizerInf(RAND_INF_COWS_MILKED_JABU_JABUS_BELLY_MQ_COW));
 
     // clang-format-on
+    registered = true;
```

Comment:
Just a personal preference, but I think it would be better to have these `registered = true` just after the early return so that it keeps it all together.

Reply:
I suppose it doesn't make much difference one way or the other, so I might as well move them yeah.

Reply:
Fixed in b89478d